### PR TITLE
v0.5.0

### PR DESCRIPTION
**Pull request by michalstolarczyk**
todo: 
- [x] [release `ubiquerg`](https://github.com/pepkit/ubiquerg/pull/23)
- [x] [release `yacman`](https://github.com/databio/yacman/pull/17)
- [x] version
- [x] release date
- [x] changelog

### CODE REVIEW

**Comment by nsheff** [comment]
how close is this to being ready to go?

File: refgenconf/helpers.py [hunk 1]
```diff
@@ -33,3 +33,33 @@ def unbound_env_vars(path):
...
+        if isinstance(key, unicode):
+            key = key.encode('utf-8')
+        if isinstance(value, unicode):
+            value = value.encode('utf-8')
+        elif isinstance(value, list):
+            value = _asciify_list(value)
+        elif isinstance(value, dict):
+            value = asciify_dict(value)
+        ret[key] = value
+    return ret
```
Comment (nsheff) [inline refgenconf/helpers.py]: are these better put into ubiquerg?

**Comment by michalstolarczyk** [comment]
It's been ready to go for some time now. I'm just fixing some minor bugs that come up in testing

**Comment by nsheff** [comment]
@vreuter if you want to take a quick look, please do so, as we're approaching release now.

**Comment by nsheff** [comment]
ok can you update the checkboxes then? Let's discuss briefly when you get in and then I'm ready to go.

File: refgenconf/helpers.py (hunk 1 above)
Comment (michalstolarczyk) [reply refgenconf/helpers.py]: good point. These are useful for every tool that fetches remote JSON-formatted data and supports both py2 and py3

File: refgenconf/exceptions.py [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 import abc
 
-__all__ = ["DownloadJsonError", "GenomeConfigFormatError", "MissingAssetError",
-           "MissingConfigDataError", "MissingGenomeError",
+__all__ = ["DownloadJsonError", "GenomeConfigFormatError", "MissingAssetError", "MissingRecipeError",
```
Comment (vreuter) [inline refgenconf/exceptions.py]: Nice type naming!

**Comment by nsheff** [comment]
does this require yacman 0.6.2?

**Comment by nsheff** [comment]
> ok can you update the checkboxes then? Let's discuss briefly when you get in and then I'm ready to go.

@michalstolarczyk can you update the changelog? and make sure the other checkboxes are ready to go? I was going to release this but realized it's not ready yet

**Comment by michalstolarczyk** [comment]
it's ready to go from my point of view. You can go ahead and make the release if there are no usage problems.

**Pull request merged**